db/hints/sync_point.cc

```diff
@@ -109,33 +145,45 @@ sync_point sync_point::decode(sstring_view s) {
         throw std::runtime_error(format("Unsupported sync point format version: {}", int(version)));
     }
 
-    sync_point_v1 v1 = ser::serializer<sync_point_v1>::read(in);
+    if (version == 2) {
+        sync_point_v1 v1 = ser::serializer<sync_point_v1>::read(in);
+
+        return sync_point{
+            v1.host_id,
+                decode_one_type_v1(v1.shard_count, v1.regular_sp),
+                decode_one_type_v1(v1.shard_count, v1.mv_sp),
+        };
```

Comment:
Not sure I saw this style of indentation before. I think in case of struct initialization we usually indent all lines within the curly braces just once.

Reply:
fixed